### ISSUE TITLE
[Crash]: Railcraft creosote oil causing a crash

### ISSUE THREAD

**Issue by fasttrak48** [AllTheMods/ATM-9#1510]
### Possible Fixes

Yes

### Modpack Version

0.2.53

### What happened?

Completing the quest for coal coke and creosote oil resulted in a game crash. I had two coke ovens being fed coal blocks by Refined Storage exporters. The creosote was being collected into a single Black Hole Tank through Pipe's fluid transfer. While having a stack of coal coke in my inventory I used a bucket to scoop up some creosote from the Black Hole Tank expecting the quest marker completed to come up but no bueno. Check the quest and it shows two potential creosote oils to complete, I check the bucket in inventory and it is a Railcraft creosote. Not sure what is going on I go to replace that bucket of oil back into the Black Hole Tank and the game crashes. At this point, upon restarting the game it just crashes with an entity error from Railcraft.

[Railcraft crash.txt](https://github.com/AllTheMods/ATM-9/files/14630962/Railcraft.crash.txt)

[latest.log](https://github.com/AllTheMods/ATM-9/files/14630968/latest.log)

### Crash Log

[Railcraft crash.txt](https://github.com/AllTheMods/ATM-9/files/14630962/Railcraft.crash.txt)

### Latest.Log

[latest.log](https://github.com/AllTheMods/ATM-9/files/14630968/latest.log)

**Comment by fasttrak48** [comment]
No backup and hoping someone might have a suggestion to get the game loaded. Would deleting Railcraft from the mods possibly allow me to get logged in or does that cause other issues?

**Comment by fasttrak48** [comment]
I actually spent some time learning about the mods and what actually happened with Railcraft Reborn. The mod is working correctly and I did not realize there were two separate coke ovens, one from Railcraft Reborn and one from Immersive Engineering. I was getting the correct liquid because I built two RR Coke ovens and not Immersive. There is a crash bug, if you try and put a RR bucket of creosote back into a black hole tank it will crash the game but seriously low priority. This thread should be closed.
